front-end/src/components/auth/Register.js

```diff
@@ -89,7 +91,16 @@ class Register extends Component {
                 />
                 <span className="red-text">{errors.email}</span>
 
-               <TextField id='password' variant='outlined' label='Password' type='password'
+              <TextField id='username' variant='outlined' label='Username'
```

Comment:
for some reason it seems like the text box isn't turning red when there's an error, but i have no idea why